src/apolo_app_types/helm/apps/superset.py

```diff
@@ -46,23 +74,36 @@ async def gen_extra_values(
         # Get base values
         values = await gen_extra_values(
             apolo_client=self.client,
-            preset_type=input_.preset,
+            preset_type=input_.web_config.preset,
             ingress_http=input_.ingress_http,
-            # ingress_grpc=input_.ingress_grpc,
             namespace=namespace,
         )
+        worker_values = await gen_extra_values(
```

Comment:
while testing please ensure we mark all pods with the corresponding preset labels, etc.